docs/docs/fundamentals/events.md

```diff
@@ -14,6 +14,21 @@ To show how event handling is done in Reanimated we are going to lead you step b
 
 Reanimated integrates tightly with the [react-native-gesture-handler](https://docs.swmansion.com/react-native-gesture-handler/) package for the ability to define performant gesture-based interactions. We explain the library's APIs whenever we use bits of it in our examples, however, if you'd like to learn more about the gesture-handler outside of the context of reanimated, please visit the documentation website [here](https://docs.swmansion.com/react-native-gesture-handler/).
 
+
+For Android devices, be sure to wrap your app entry point with `<GestureHandlerRootView>` component from `react-native-gesture-handler` library to capture events properly.
```

Comment:
```suggestion
For Android OS, be sure to wrap your app entry point with `<GestureHandlerRootView>` component from `react-native-gesture-handler` library to capture events properly.
```
Maybe we could modify it to prevent any misunderstanding that it only pertains to physical devices and not emulators as well.